.backportrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "upstream": "elastic/beats",
-  "branches": [ { "name": "7.x", "checked": true }, "7.10", "7.9", "7.8", "7.7"],
+  "branches": [ { "name": "7.x", "checked": true }, "7.10" ],
```

Comment:
+1

Shall we keep in 7.9 until 7.10 has been released?

Reply:
7.10.0 will be released in the next days